zubhub_frontend/zubhub/public/locales/hi/translation.json

```diff
@@ -292,6 +292,18 @@
       "primary": "प्रोजेक्ट बनाएं",
       "secondary": "हमें अपनी परियोजना के बारे में बताएं!"
     },
+    "selectMode": {
+      "description": "चुनें कि यह किस प्रकार का प्रोजेक्ट है, यदि आपने अकेले प्रोजेक्ट पर काम किया है तो व्यक्तिगत प्रोजेक्ट चुनें, यदि आपने अपने प्रोजेक्ट पर अन्य रचनाकारों के साथ काम किया है तो टीम प्रोजेक्ट चुनें।",
+      "personalMode": {
+        "title": "व्यक्तिगत परियोजना",
```

Comment:
Fine doing translations for now, but we would need to rework them later anyways since they are not that accurate :) Thanks for efforts and consistency considering this is not your first language 😄

Reply:
Thank you :D not giving up on google translate